elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -84,12 +84,19 @@ public function filter__ep_index_name( $index_name, $blog_id, $indexables ) {
 
 	public function filter__ep_do_intercept_request( $request, $query, $args, $failures ) {
 		$fallback_error = new \WP_Error( 'vip-elasticsearch-upstream-request-failed', 'There was an error connecting to the upstream Elasticsearch server' );
+		$timeout = 1;
 
 		// TEMP - currently ES server is using a self signed cert during the testing phase...that'll be changed in the near
 		// future, at which time we can remove this
 		$args['sslverify'] = false;
-	
-		$request = vip_safe_wp_remote_request( $query['url'], $fallback_error, 3, 1, 20, $args );
+
+		// If query url ends with '_bulk'
+		if ( 0 === substr_compare( $query[ 'url' ], '_bulk', -5 ) ) {
+			// Bulk index request so increase timeout
+			$timeout = 3;
```

Comment:
```suggestion
			$timeout = 5;
```

From my testing, `5` seems to be a good upper limit (I've seen timeouts at `3`).

Reply:
We'll need to change `vip_safe_wp_remote_request()` for this too work. It's currently set to limit `timeout` to a max of 3 seconds

Reply:
Changed in ad0f04966b6984075dec2f0de02581009339a05e

Reply:
Crud, you're right...let's see how it goes at 3 for now and revisit later.

Reply:
Oh just saw you bumped it up in vip_safe_wp_remote_request) - I think that'll be fine. It'll let sites connect to slower APIs on the frontend, but it's not _that_ much different, and the fallback stuff will still happen when there are multiple requests for a slow resource back to back. cc @mjangda for additional thoughts.

Reply:
@mjangda Do you have any thoughts on ^?

Reply:
Only concern would sites that are actively hitting that 3s or higher timeout. If we bump their sites will slow down even further. However, if that's not a problem we're seeing 🚢 

As as aside, it's probably worth throwing a `_doing_it_wrong` when someone passes in a value that's > 5.

Reply:
> As as aside, it's probably worth throwing a _doing_it_wrong when someone passes in a value that's > 5.

Added in 51e7c4de5cd21e21deb5f49e19885f4182ea40eb